Detectors/MUON/MCH/Raw/ElecMap/src/testElectronicMapper.cxx

```diff
@@ -17,12 +17,14 @@
 #include <boost/test/unit_test.hpp>
 
 #include "MCHRawElecMap/Mapper.h"
-#include <fmt/format.h>
-#include <set>
+
+#include "dslist.h"
+#include <array>
 #include <boost/mpl/list.hpp>
+#include <fmt/format.h>
 #include <gsl/span>
-#include <array>
-#include "dslist.h"
+#include <set>
+#include <unordered_map>
```

Comment:
I suspect the include of unordered_map is not needed anymore

Reply:
indeed